lms/djangoapps/grades/migrations/0011_null_edited_time.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('grades', '0010_auto_20170112_1156'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='persistentcoursegrade',
+            name='course_edited_timestamp',
+            field=models.DateTimeField(null=True, verbose_name='Last content edit timestamp', blank=True),
+        ),
+        migrations.AlterField(
+            model_name='persistentsubsectiongrade',
+            name='course_version',
+            field=models.CharField(max_length=255, verbose_name='Guid of latest course version', blank=True),
```

Comment:
I only touched this field to make our usage of capitalized unicode consistent in `grades/models.py`, it doesn't seem to affect the SQL being run at all though:

```
edxapp@precise64:~/edx-platform$ ./manage.py lms --settings=devstack sqlmigrate grades 0011
BEGIN;
ALTER TABLE `grades_persistentcoursegrade` MODIFY `course_edited_timestamp` datetime(6) NULL;
ALTER TABLE `grades_persistentsubsectiongrade` MODIFY `subtree_edited_timestamp` datetime(6) NULL;

COMMIT;
```